src/activity_item_handling.cpp

```diff
@@ -2655,6 +2839,61 @@ static bool generic_multi_activity_do( player &p, const activity_id &act_id,
         }
         construction_activity( p, zone, src_loc, act_info, list_constructions, act_id );
         return false;
+    } else if( reason == CAN_DO_CRAFT ) {
+        std::vector<item_location> crafts;
+        auto items_at_furn = g->m.i_at( src_loc );
+        for( item &it : items_at_furn ) {
```

Comment:
Don't declare `items_at_furn` variable as it isn't used anywhere else.
```suggestion
        for( item &it : g->m.i_at( src_loc ) ) {
```